instrumentation/mongo/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History: opentelemetry-instrumentation-mongo
 
+### v0.21.0 / 2022-10-04
```

Comment:
```suggestion
### v0.22.0 / 2022-10-04
```

Reply:
This PR has already bumped otel-inst-mongo from 0.20.0 to 0.21.0 (the [commit](https://github.com/open-telemetry/opentelemetry-ruby-contrib/commit/06b06242fd9ac16bab74e98eee0853bb9b91adad) had a `feat:` preamble! 👍🏻).

What led you to bump it once more to 0.22.0?

Reply:
A mistake by me